### PR TITLE
Heroku deploy

### PR DESCRIPTION
**Pull request by alanpeabody**

### CODE REVIEW

**Comment by jayzes** [comment]
We should probably strip out the key in that cassette, no?

**Comment by alanpeabody** [comment]
It is the key from their API documentation, not one of ours. Not that I think they are really sensitive.

File: lib/ello_click/affiliate.ex [hunk 1]
```diff
@@ -10,13 +10,21 @@ defmodule ElloClick.Affiliate do
   end
 
   defp affiliate(conn) do
-    conn.request_path
+    conn
+    |> full_url
```
Comment (Justin-Holmes) [inline lib/ello_click/affiliate.ex:14]: Would:

```
full_url(conn)
|> clean
...
```

work too?

**Comment by jayzes** [comment]
Ah, perfect. Nevermind in that case, then.

File: lib/ello_click/affiliate.ex (hunk 1 above)
Comment (alanpeabody) [inline lib/ello_click/affiliate.ex:14]: Yup, but it is generally preferred to start pipelines with a "pure" value rather then a function call. It doesn't _always_ make sense to though.

**Pull request merged**